docs/docs/agents/overview.md

```diff
@@ -53,3 +53,136 @@ The high-level components are organized into several packages, each with a speci
 | `langmem`                                  | Agent memory management: [**short-term and long-term**](./memory.md)        | `pip install -U langmem`                |
 | `agentevals`                               | Utilities to [**evaluate agent performance**](./evals.md)                   | `pip install -U agentevals`             |
 
+## Agent Graph Visualizer
+
+We've created the following tool to help users visualize the graph generated by `create_react_agent` and view the outline of the corresponding code.
+It allows you to explore the infrastructure of the agent as defined by the presence of:
+
+* `tools`: A list of tools (functions, APIs, or other callable objects) that the agent can use to perform tasks.
```

Comment:
`tools`, `pre_model_hook`, `post_model_hook` and `response_format` should all link to relevant doc pages